biggus/tests/unit/test__Elementwise.py

```diff
@@ -64,5 +73,44 @@ def test_power(self):
         self._test(np.power, ma.power)
 
 
+class Test_single_argument_dtype(unittest.TestCase):
+    def setUp(self):
+        dtypes = []
+        for dtype in ['int', 'uint', 'float']:
+            dtypes.extend(np.sctypes[dtype])
+        self.dtypes = dtypes
+
+    def dtype_after_of(self, dtype, op):
+        return op(np.ones(1, dtype=dtype)).dtype
+
+    def _test(self, *ops):
+        for dtype in self.dtypes:
+            a1 = np.ones(1, dtype=dtype)
+            actual = Elementwise(a1, None, *ops)
+            expected_dtype = self.dtype_after_of(dtype, ops[0])
+            self.assertEqual(actual.dtype, expected_dtype)
+
+    def test_abs(self):
+        self._test(np.abs, ma.abs)
+
+    def test_rint(self):
+        self._test(np.rint)
+
+    def test_sign(self):
+        self._test(np.sign)
+
+    def test_square(self):
```

Comment:
Why these ufuncs in particular? Do they exhibit specific dtype behaviour that is worth explicitly testing for, or were they chosen as a random sample set?